docs/cli/linear.rst

```diff
@@ -9,19 +70,27 @@ For training, use
 .. code-block:: bash
 
     python3 main.py --config CONFIG_PATH \
-                    --linear [--liblinear_options LIBLINEAR_OPTIONS]
+                    --linear \
+                    --liblinear_options LIBLINEAR_OPTIONS \
+                    --linear_technique MULTILABEL_TECHNIQUE
+
+- **config**: configure parameters in a yaml file.  A validation set is not needed because the program may split the training set for internal validation. If specified, it will be ignored.
 
-- **config**: configure parameters in a yaml file.
+The linear classifiers are based on `LIBLINEAR <https://www.csie.ntu.edu.tw/~cjlin/liblinear/>`_, and its options may be specified.
 
-The linear classifiers are based on LIBLINEAR, and its options may be specified.
+- **linear**: When this option exists, it is set to True such that the linear classifiers will be run. Otherwise it is set to False by default such that the neural netowrk module will be executed and the program will terminate if the neural network config is not specified.
```

Comment:
```suggestion
- **linear**: If this option exists, it is set to True such that the linear classifiers will be run. Otherwise it is set to False by default such that the neural network module will be executed and the program will terminate if the neural network config is not given.
```